middleware/helpers.js

```diff
@@ -50,3 +50,35 @@ module.exports.getMaxDate = () => {
 
   return `${year}-${month}-${day}`;
 };
+
+function checkMediaLink(mediaLink, linkRegex) {
+  const match = mediaLink.match(linkRegex);
+  return match && match[3];
+}
+
+module.exports.mediaLinkValidation = (facebook, twitter, instagram, tumblr) => {
```

Comment:
I would prefer if the function arguments was an object that would be destructured for use later to minimize possibility of mistakes when passing function arguments

```
module.exports.mediaLinkValidation = ({ facebook, twitter, instagram, tumblr }) => {
...
```